src/lib/tabs/tab-header.ts

```diff
@@ -387,6 +388,7 @@ export class MdTabHeader extends _MdTabHeaderMixinBase
    * This is an expensive call that forces a layout reflow to compute box and scroll metrics and
    * should be called sparingly.
    */
+  @HostListener('window:resize')
```

Comment:
We don't really use the `@HostListener` anywhere. Can you switch to using the `host` property on the component definition instead>